app/javascript/packages/document-capture/components/form-steps.jsx

```diff
@@ -46,6 +46,7 @@ import useForceRender from '../hooks/use-force-render';
  * @prop {string} title Step title, shown as heading.
  * @prop {import('react').FC<FormStepComponentProps<Record<string,any>>>} form Step form component.
  * @prop {import('react').FC=} footer Optional step footer component.
+ * @prop {(Object)=>boolean=} validator Optional function to validate values for the step
```

Comment:
It's discouraged to use the `Object` type in TypeScript:

>Don’t ever use the types Number, String, Boolean, Symbol, or Object These types refer to non-primitive boxed objects that are almost never used appropriately in JavaScript code.

https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#general-types

As mentioned in the documentation, you can use the lower-case `object`:

```suggestion
 * @prop {(object)=>boolean=} validator Optional function to validate values for the step
```